<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccesGranted</class>
 <widget class="QWidget" name="AccesGranted">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_key">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>781</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(173, 173, 177);
color: rgb(255, 255, 255);
border-color: rgb(94, 80, 93);
border-style: outset;
border-width: 3px;
border-color: rgb(10, 10, 10);
color: rgb(241, 242, 243);
   </string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_key1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_key2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_key3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_key4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_key9">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_key8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_key7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_key6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_key0">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>270</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_key5">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>160</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_keyCancel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>390</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_keyBack">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   background-color: red;
    border-style: outset;
    border-width: 10px;
    border-radius: 20px;
    border-color: rgb(241, 247, 243);
	color: rgb(241, 242, 243);
    font: bold 14px;</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
